tests/repository_data/repository/1.0.0/role1.json

```diff
@@ -34,7 +34,7 @@
    ]
   },
   "expires": "2030-01-01T00:00:00Z",
-  "spec_version": "1.0.0",
```

Comment:
Please keep in mind that you can't just replace the value of any of the fields in the `signed` dictionary without re-signing the files again or the files will fail verification.

Reply:
This I believe was made as a temporary change rather and the original state of `repository_data/repository` was brought back again in `774a039`. 

That being said, all of the metadata was copied over inside the `repository_data/TAP 14/1.0.0` & `repository_data/TAP 14/2.0.0` folders and metadata inside the `2.0.0` had the `"spec_version"` changed to `2.0.0` in a similar manner (just to show that this was metadata belonging to 2.0.0).
That seems like something I'll have to take a look at, thanks.